ODIN_II/SRC/read_blif.cpp

```diff
@@ -469,8 +470,25 @@ char* search_clock_name(FILE* file)
 	file_line_number = last_line;
 	fsetpos(file,&pos);
 
-	if (found) return input_names[0];
-	else       return vtr::strdup(DEFAULT_CLOCK_NAME);
+	if (found)
+	{
+		to_return = input_names[0];
```

Comment:
should you free all but the the input_name at index 0?

Reply:
The only way that found is flipped to true is if input_names_count is 1, so logically, input_names[0] is the only string in the array, so there's no need to free in this block